components/merkledb/build.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use exonum_build::ProtobufGenerator;
```

Comment:
Nit: won't these imports be marked as unused if the crate is built without Protobuf?

Reply:
@pavel-mukhanov I'd move this import into `gen_proto_files` function.